eng/pipelines/runtimelab.yml

```diff
@@ -151,6 +171,45 @@ stages:
           dependsOn:
           - build_windows_x64_Release_
 
+    #
+    # Libraries Debug Test Execution against a release coreclr runtime
+    #
+    - ${{ if eq(variables.isOfficialBuild, false) }}:
+      - template: /eng/pipelines/common/platform-matrix.yml
+        parameters:
+          jobTemplate: /eng/pipelines/libraries/run-test-job.yml
+          buildConfig: Debug
+          platforms:
+          - Linux_x64
+          helixQueuesTemplate: /eng/pipelines/libraries/helix-queues-setup.yml
+          jobParameters:
+            isFullMatrix: false
+            isOfficialBuild: false
+            testScope: innerloop
+            liveRuntimeBuildConfig: Release
+            dependsOnTestBuildConfiguration: Release
+            dependsOnTestArchitecture: x64
+            dependsOn:
+            - build_Linux_x64_Debug_
+
+    - ${{ if eq(variables.isOfficialBuild, false) }}:
```

Comment:
You can remove this line as yml is indentation based so it will use the condition from line: 177.